testsuite/tests/properties/abstract_rt/expected_concrete_syntax.lkt

```diff
@@ -0,0 +1,19 @@
+import lexer_example
+@with_lexer(foo_lexer)
+grammar foo_grammar {
+    @main_rule main_rule <- list*(or(Example("example") | Error("error")))
+
+}
+
+@abstract class FooNode implements Node[FooNode] {
+
+    @export @not_implemented fun prop(): Int
```

Comment:
Not fond of the name, but I don't have anything better right now...

Reply:
I wonder if we shouldn't drop this feature considering the alternative:

```
    @export fun prop(): Int = raise PropertyError("prop not implemented on" & Self.image)
```

PM, @Roldak, wdyt ?

Reply:
I’m completely ok to drop this feature: it probably made sense back when we didn’t have raise expressions, but I think we could get rid of it nowadays.

(otherwise I would be ok to rename it, but in another commit given that this PR does not introduce it)

Reply:
As discussed live, I’ve removed @not_implemented in dsl_unparse (nothing to do left in Lkt in that domain)